app/visualizers/view/controller.js

```diff
@@ -1,5 +1,6 @@
 ﻿app.controller('visualizerController', ['$scope', '$q', '$uibModal', '$sce', '$window', 'toastr', 'visualizerService', function ($scope, $q, $uibModal, $sce, $window, toastr, visualizerService) {
 
+    console.log('visualizerController', siteRoot);
```

Comment:
Can we remove this console log please.

Reply:
Thank you

Reply:
Looks like this was removed in the latest commit.  I'll go ahead and mark this conversation as resolved.